bin/xspec.bat

```diff
@@ -372,6 +385,48 @@ rem ## compile the suite #######################################################
 rem ##
 rem
 
+rem # get URI to Schematron file and phase from the SchUT file
+if defined SCHEMATRON (

```

Comment:
I see a series of `if defined SCHEMATRON` is inevitable, but it looks error-prone. It would be nice to do `goto` and skip the bulk of the lines. That will
* simplify the code
* prevent mistakes like the current test [failure](https://ci.appveyor.com/project/xspec/xspec/build/1.0.150/job/sico8n9i8ogkdk9r#L108)

Though `goto` is in general not a good habit, I think for this particular case the end justifies the means.